src/ConstructiveSolidGeometry/Transformations.jl

```diff
@@ -6,8 +6,12 @@ end
 in(p::CartesianPoint, g::ScaledGeometry) = in(CartesianPoint(g.inv_s .* p), g.p)
 scale(g::AbstractGeometry{T}, s::SVector{3,T}) where {T} = ScaledGeometry(g, s)
 scale(g::ScaledGeometry{T}, s::SVector{3,T}) where {T} = ScaledGeometry(g.p, inv.(g.inv_s) .* s)
-(*)(g::AbstractGeometry{T}, s::SVector{3,T}) where {T} = scale(g, s)
-get_plot_points(sg::ScaledGeometry{T}) where {T} = get_plot_points(sg.p) * inv.(sg.inv_s)
+#(*)(g::AbstractGeometry{T}, s::SVector{3,T}) where {T} = scale(g, s)
+function get_plot_points(sg::ScaledGeometry{T}) where {T}
+    pp = get_plot_points(sg.p)
+    scale!(pp, inv.(sg.inv_s))
+    pp
+end
```

Comment:
I would define the `translate!`, `scale!` and `rotate!` methods such, that they return the vector again. 
Then, you can write here:
```julia
get_plot_points(sg::ScaledGeometry) = scale!(get_plot_points(sg.p), inv.(sg.inv_s))
```
This might also come in handy in other situations.